mautrix_telegram/config.py

```diff
@@ -197,6 +197,10 @@ def do_update(self, helper: ConfigUpdateHelper) -> None:
 
         copy("bridge.filter.mode")
         copy("bridge.filter.list")
+        if "bridge.filter.users" not in self:
+            base["bridge.filter.users"] = True
+        else:
+            copy("bridge.filter.users")
```

Comment:
Missing fields are already handled by `copy`

```suggestion
        copy("bridge.filter.users")
```

Reply:
Does it provide the correct default, though...?

Or does it just set it to `None`?